client/tests/integration/restart_peer.rs

```diff
@@ -24,7 +24,13 @@ fn restarted_peer_should_have_the_same_asset_amount() -> Result<()> {
 
     // Given
     let rt = Runtime::test();
-    rt.block_on(peer.start_with_config_permissions_dir(configuration.clone(), AllowAll, &temp_dir));
+    rt.block_on(peer.start_with_config_permissions_dir(
+        configuration.clone(),
+        GenesisNetwork::test(true),
+        AllowAll,
```

Comment:
https://github.com/hyperledger/iroha/issues/2199

Refers to this. The permissions need to be specified inside the genesis block, not hard-coded in the init process.